data/profile.d/fedora-kde-mobile.conf

```diff
@@ -0,0 +1,12 @@
+# Anaconda configuration file for Fedora KDE Plasma Mobile spin.
+
+[Profile]
```

Comment:
So this is just preparatory work, right ? As the profile currently does not do anything.

Reply:
It should inherit from `fedora-kde` and use all of its settings.